cmd/client/cmd/upload.go

```diff
@@ -52,9 +52,15 @@ func newUploadCmd() *cobra.Command {
 				return fmt.Errorf("upload file: get cli parameter 'file': %w", err)
 			}
 
-			original, err := cmd.Flags().GetBool("original")
-			if err != nil {
-				return fmt.Errorf("upload file: get cli parameter 'original': %w", err)
+			var original *bool
+
+			if cmd.Flags().Changed("original") {
+				v, flagErr := cmd.Flags().GetBool("original")
+				if flagErr != nil {
```

Comment:
```suggestion
				v, err := cmd.Flags().GetBool("original")
				if err != nil {
```

Reply:
That's there to avoid linter from yelling, I will add nolint then.